waku/v2/protocol/pubsub_topic.go

```diff
@@ -7,104 +7,43 @@ import (
 	"strings"
 )
 
-// Waku2PubsubTopicPrefix is the expected prefix to be used for pubsub topics
-const Waku2PubsubTopicPrefix = "/waku/2"
```

Comment:
I think it is better to leave this prefix as is and append either default pubsubTopic suffix or sharded topic suffix.
That way, in future if there is a change to this prefix it would be easier to maintian.